src/constants/localStorage.ts

```diff
@@ -0,0 +1,4 @@
+export enum LOCAL_STORAGE_KEYS {
+	// ----- CHOOSEN_WALLET ----- //
+	CHOOSEN_WALLET = '/chosenWallet',
+}
```

Comment:
@colbr @padala @domw30 @akshatmittal We will use global constants for local storage keys instead of hard coding